docs/getting-started/cli-getting-started.md

```diff
@@ -15,26 +15,26 @@ Get started with the **Stable (@lts-incremental) CLI version** quickly and easil
   - [Writing scripts](#writing-scripts)
   - [Next Steps](#next-steps)
 
-## Installing 
+## Installing
 
 Before you install Zowe CLI, download and install [Node.js and npm.](https://nodejs.org/en/download/)
 
 ### Installing Zowe CLI core
 
 ```
-npm config set @zowe:registry 
+npm config set @zowe:registry
 ```
 
 ```
 npm install @zowe/cli -g
 ```
 
-### Installing CLI plug-ins 
+### Installing CLI plug-ins
 
 The command installs the IBM CICS, IBM IMS, and IBM MQ plug-ins:
 
 ```
-zowe plugins install @zowe/cics @zowe/ims @zowe/db2 @zowe/mq
+zowe plugins install @zowe/cics @zowe/ims @zowe/db2 @zowe/mq @zowe/secure-credential-store-for-zowe-cli
```

Comment:
This may result in a conflict since there is another PR that fixes all other plugin names :)